cli/remote_build.go

```diff
@@ -415,6 +427,38 @@ func (p *RemoteCmd) traverseChanges(worktree *git.Worktree, saver *WorktreeSave)
 	return
 }
 
+func (p *RemoteCmd) skip(file string, worktree *git.Worktree) bool {
```

Comment:
Can we rename this to shouldSkip and then put the append logic outside this call?